### PR TITLE
Day planer

### PR DESCRIPTION
**Pull request by J-Massmann**

### CODE REVIEW

File: src/components/DayCard.spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import DayCard from './DayCard.js';
+import { render, screen } from '@testing-library/react';
+
+describe('DayCard', () => {
+  it('renders an article with a header and a paragrpah', () => {
```
Comment (ahohnsen) [inline src/components/DayCard.spec.js:5]: Small typo
```suggestion
  it('renders an article with a header and a paragraph', () => {
```

Comment (ahohnsen) [reply src/components/DayCard.spec.js:5]: As you are rendering a heading and a span element, maybe it is better so write her something like: "renders the day and date of a trip"

**Review by ahohnsen — APPROVED** [review]
Nice work, just a little suggestion :)

File: src/components/DayCard.spec.js (hunk 1 above)
Comment (J-Massmann) [reply src/components/DayCard.spec.js:5]: uups, that's true a don't even render a paragraph 😂

**Pull request merged**